pyathena/sqlalchemy_athena.py

```diff
@@ -211,18 +275,34 @@ def visit_create_table(self, create):
         if const:
             text += separator + "\t" + const
 
-        text += "\n)\n%s\n\n" % self.post_create_table(table)
+        text += "\n)\n%s\n\n" % self.post_create_table(create)
         return text
 
-    def post_create_table(self, table):
+    def post_create_table(self, create):
```

Comment:
Shouldn't we make this private?
```suggestion
    def _post_create_table(self, create):
```